include/install-vitasdk.sh

```diff
@@ -8,7 +8,7 @@ install_vitasdk () {
   INSTALLDIR=$1
 
   case "$(uname -s)" in
-     Darwin*)
+     Darwin*|FreeBSD*)
```

Comment:
is `*BSD*` also fine (NetBSD, OpenBSD, PC-BSD) ?

Reply:
Well, I tried in FreeNAS and FreeBSD, and it was exactly FreeBSD, so *BSD may be a better solution.